### DISCUSSION TITLE
Baggage claim runner exits with error: "error":"Exit trace for group:\napi exited with error - listen tcp <ip_address>:7788: bind: cannot assign requested address\ndebug-server exited with nil\n"

### DISCUSSION THREAD

**Discussion by RagHarnoor** [concourse/concourse#6350] [Help & Support]
## The concourse worker container doesn't register with web node because of the above mentioned error with baggage claim runner 

Our concourse setup involves web and worker concourse containers deployed on mesosphere private cloud. The worker node when deployed doesn't register with the web node and throws this error:

>{"timestamp":"2020-12-07T15:59:19.393044787Z","level":"error","source":"worker","message":"worker.baggageclaim-runner.logging-runner-exited","data":{"error":"Exit trace for group:\napi exited with error: listen tcp 171.68.197.124:7788: bind: cannot assign requested address\ndebug-server exited with nil\n","session":"9"}}
{"timestamp":"2020-12-07T15:59:19.393109363Z","level":"info","source":"worker","message":"worker.debug-runner.logging-runner-exited","data":{"session":"10"}}

## Steps to reproduce

- deploy concourse web and worker containers on mesopshere cloud 
- look at the worker logs and chck whether the worker container shows up in the list of workers available with the master.

## Expected results
Concourse worker to register with master 

## Actual results
Worker container failed to register because of baggage claim runner failing to run

## Additional context
I ensured that the ports used on the host were 7777 for Garden server, 7788 for baggage claim and 7799 for garbage collection to isolate the issue

please let me know if any other info is needed

## Triaging info

* Concourse version: 6.7.2
* Browser (if applicable):
* Did this used to work?

**Comment by taylorsilva** [comment]
As I think you saw, the worker doesn't come up because baggageclaim can't bind/listen to port 7788
```
exited with error: listen tcp 171.68.197.124:7788: bind: cannot assign requested address
```

That IP address is odd. Baggageclaim binds to `127.0.0.1` by default. `171.68.197.124` is a public IP address. Have you tried configuring baggageclaim to ues `127.0.0.1` instead or just not configure the bind ip and use the default?

Also baggageclaim is **NOT** secure on its own. Exposing it to the public internet is definitely a bad idea.
